doc/source/whatsnew/v0.21.0.txt

```diff
@@ -389,7 +389,7 @@ New Behavior:
   ---------------------------------------------------------------------------
   ValueError: Of the three parameters: start, end, and periods, exactly two must be specified
 
-  In [3]: pd.period_range(start='2017Q1', end='2017Q4', periods=6, freq='Q')  
```

Comment:
Hmm...what happened here?

Reply:
Atom stripped the trailing whitespace.

Reply:
I can add it back if you want.

Reply:
No, make the change by all means!  If this was a Python file, `flake8` would have been unhappy. 😄